websites/S/Stats.fm/presence.ts

```diff
@@ -0,0 +1,81 @@
+const presence = new Presence({
+		clientId: "973027832307535952"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "stats_fm",
+			startTimestamp: browsingTimestamp
+		},
+		{ pathname } = window.location;
+	switch (true) {
+		case pathname === "/": {
+			presenceData.details = "Browsing the main page";
+			break;
+		}
+		case pathname.includes("/album/"): {
+			presenceData.details = "Viewing an album";
+			presenceData.state = document.querySelector("h1.text-center").textContent;
+			presenceData.buttons = [
+				{
+					label: "View Album",
+					url: location.href
+				}
+			];
+			break;
+		}
+		case pathname.includes("/artist/"): {
+			presenceData.details = "Viewing an artist";
+			presenceData.state = document.querySelector("h1.text-center").textContent;
+			presenceData.buttons = [
+				{
+					label: "View Artist",
+					url: location.href
+				}
+			];
+			break;
+		}
+		case pathname.includes("/track/"): {
+			presenceData.details = "Viewing stats for a track";
+			presenceData.state = document.querySelector("h1.text-center").textContent;
+			presenceData.buttons = [
+				{
+					label: "View Track",
+					url: location.href
+				}
+			];
+			break;
+		}
+		case pathname.includes("/beta"): {
+			presenceData.details = "Viewing Beta informations";
```

Comment:
Feel free to pick one
```suggestion
			presenceData.details = "Viewing Beta information";
```
or
```suggestion
			presenceData.details = "Viewing information about the beta";
```